osu.Game/Beatmaps/Drawables/GroupedDifficultyIcon.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Rulesets;
+using osuTK.Graphics;
+
+namespace osu.Game.Beatmaps.Drawables
+{
+    public class GroupedDifficultyIcon : DifficultyIcon
```

Comment:
could we please at very least xmldoc classes describing what they are when the name is not enough to do so?

Reply:
oh sorry, i forgot to add xmldocs